src/EnergyPlus/OutputReportTabular.hh

```diff
@@ -970,6 +973,11 @@ struct OutputReportTabularData : BaseGlobalStruct
 {
 
     OutputReportTabular::UnitsStyle unitsStyle = OutputReportTabular::UnitsStyle::None;
+    bool ip() const
+    {
+        return this->unitsStyle == OutputReportTabular::UnitsStyle::InchPound ||
+               this->unitsStyle == OutputReportTabular::UnitsStyle::InchPoundExceptElectricity;
+    }
```

Comment:
@jcyuan2020 this is what I meant.  Add a member method to the existing ort-based structure, not adding a whole new struct.